src/providers/apolloProvider.ts

```diff
@@ -170,16 +176,14 @@ export class ArtificialApollo {
       });
 
       if (result && result.data) {
+        this.throwError.cancel();
```

Comment:
This cancels the debounce if apollo resolves.

This commonly happens when config changes as we get 2 events.  First event clears the file.  Second event fills it in with the new config.  This suppresses errors until that second event can resolve.